Tests/StreamChatTests/WebSocketClient/WebSocketClient_Tests.swift

```diff
@@ -99,7 +99,7 @@ final class WebSocketClient_Tests: XCTestCase {
         // Save currently existed engine.
         let oldEngine = webSocketClient.engine
         // Disconnect the client.
-        webSocketClient.disconnect()
+        webSocketClient.disconnect {}
```

Comment:
If this is async now, don't we need to wait for an expectation here as well? 🤔

Reply:
It's fine, what matters in this test is 2nd `connect` call which should not recreate the engine